docs/cli-usage.md

```diff
@@ -0,0 +1,96 @@
+## Using Kubernetes from the command line
```

Comment:
I'd put resize, label updates, and annotation updates into the category of common narrow imperative updates. As mentioned somewhere(?), we'll want to optionally make these safe(r) by supporting preconditions based on the current value and resourceVersion. If we add "stop" and/or "suspend" bits to each object for graceful termination and/or deferred enablement, modification of those bits should be similarly supported.

We also want to support the following flavors of more general updates:
* general surgical imperative update: Update just the fields specified on the command line, optionally protected via preconditions, as above.
* safe update: update the full resource, guarded by resourceVersion precondition (and perhaps selected value-based preconditions)
* forced update: update the full resource, blowing away the previous Spec without preconditions
* diff/dryrun (`diff`): Compare new config with current Spec
* reconcile: Merge user-provided fields with current Spec. (Keep track of user-provided fields using an annotation.)